.mvn/wrapper/maven-wrapper.properties

```diff
@@ -6,7 +6,7 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#   https://www.apache.org/licenses/LICENSE-2.0
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why this change?

Reply:
I don't know :) All the changes are produced by the wrapper goal. My guess is that it's some alignment.

Reply:
@guylabs see here: https://github.com/apache/maven-wrapper/commit/8a1ded6cc4de5b2c9e238efe84ae122fdd8db050